src/mlpack/tests/CMakeLists.txt

```
@@ -31,7 +31,7 @@ add_executable(mlpack_test
   det_test.cpp
   distribution_test.cpp
   drusilla_select_test.cpp
-  emst_test.cpp
+  emst_test.cpp 
```

```suggestion
  emst_test.cpp
```
No need for an extra space. :)